docs/development.md

```diff
@@ -737,6 +737,38 @@ and IO errors are difficult to simulate in C.) Code coverage statistics are
 automatically tracked using [CodeCov](https://codecov.io/gh/tskit-dev/tskit/).
 
 
+### Viewing coverage reports
+
+To generate and view coverage reports for the C tests locally:
+
+Compile with coverage enabled:
+   ```bash
+   $ meson build-gcc c -D b_coverage=true
```

Comment:
The -gcc bit here will likely confuse people (who may not be using GCC). Just use ``meson build -D b_coverage=tru ``.

Also simpler to say "do this in the c directory``?

Also I think meson raises a warning if you use it like this now, isn't there a new recommened way of calling it?